### PR TITLE
Purchase summary

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?
Layout adjustments to `Summary` component.

#### Screenshots or example usage

screen shot 2019-01-29 at 8 39 08 pm

### CODE REVIEW

File: react/components/Header/Summary.js [hunk 1]
```diff
@@ -25,56 +26,61 @@ const OrderSummary = ({ data, intl }) => {
...
-              })}
-            </p>
-            <p className="t-heading-4">
-              <TranslateEstimate
-                shippingEstimate={longestDeliveryEstimate.shippingEstimate}
-              />
-            </p>
-            <small className="t-small c-muted-2">
+          <article className="center flex flex-column">
+            <div>
```
Comment (kevinch) [inline react/components/Header/Summary.js]: Not sure this `div` is useful.

File: react/components/Header/Summary.js [hunk 2]
```diff
@@ -25,56 +26,61 @@ const OrderSummary = ({ data, intl }) => {
...
-                shippings: totalDeliveries.length,
-              })}
-            </p>
-            <p className="t-heading-4">
-              <TranslateEstimate
-                shippingEstimate={longestDeliveryEstimate.shippingEstimate}
-              />
-            </p>
-            <small className="t-small c-muted-2">
+          <article className="center flex flex-column">
```
Comment (kevinch) [inline react/components/Header/Summary.js]: Useless `flex` & `flex-column` classes. Even if the direct child `div` is removed, `p` elements are block elements, therefore they will stack vertically.

File: react/components/Header/Summary.js [hunk 3]
```diff
@@ -25,56 +26,61 @@ const OrderSummary = ({ data, intl }) => {
...
               {intlMessage(intl, 'summary.shipping.address', {
                 addressStreet: totalDeliveries[0].address.street,
                 addressNumber: totalDeliveries[0].address.number,
               })}
-            </small>
+            </p>
           </article>
-          <article className="mb8 center">
-            <p className="t-heading-4 tc">
+          <article className="center flex flex-column">
```
Comment (kevinch) [inline react/components/Header/Summary.js]: Same here.

File: react/components/Header/Summary.js (hunk 1 above)
Comment (victorhmp) [reply react/components/Header/Summary.js]: It slipped through... removed :)

File: react/components/Header/Summary.js (hunk 2 above)
Comment (victorhmp) [reply react/components/Header/Summary.js]: Yep, removed them.

File: react/components/Header/Summary.js (hunk 3 above)
Comment (victorhmp) [reply react/components/Header/Summary.js]: Removed.

**Pull request merged**